internal/nodes/avi_model_routeingr_hostname_shard.go

```diff
@@ -93,13 +93,13 @@ func HostNameShardAndPublish(objType, objname, namespace, key string, fullsync b
 	hostsMap := make(map[string]*objects.RouteIngrhost)
 
 	if lib.IsEvhEnabled() {
+		// delete stale data
+		DeleteStaleDataForEvh(routeIgrObj, key, &modelList, Storedhosts, hostsMap)
```

Comment:
Calling DeleteStaleDataForEvh before processing would mean deletion of all objects from model, every time we get an update and then building the model again. 
What was the the issue if DeleteStaleDataForEvh is called after generating the models ?